test/migrators/division/always_division.hrx

```diff
@@ -49,4 +64,9 @@ a {
   l: divide(3, $x) > 1;
   m: divide(3, $x) <= 1;
   n: divide(3, $x) >= 1;
+
+  // Function calls
+  o: identity(divide(6px, 3px));
+  p: rgba(10, 20, divide(30, 2), 0.5);
+  q: rgb(10 20 divide(30, 2) / 0.5);
```

Comment:
This actually isn't a valid translation; because `divide()` isn't a literal number, this `/` is *always* interpreted as division, which means the metadata about what the right-hand side used to be gets lost before it reaches `rgb()`. (This is the kind of nasty confusing edge case that makes the current `/` semantics so awful.)

I think the ideal translation here is probably `rgb(10, 20, divide(30, 2), 0.5)`, but it's also reasonable to do `rgb(10 20 slash-list(divide(30, 2), 0.5))` if that's easier.